.github/workflows/deploy-documentation.yml

```diff
@@ -29,11 +29,11 @@ jobs:
       - name: Unzip Documentation Results
         run: |
           unzip docs_results.zip
-          rm -r docs_results/.results_docs/html/.doctrees
```

Comment:
See failure here: https://github.com/tektronix/tm_devices/actions/runs/6674741459/job/18141949437 for the reason for the change.